syft/frameworks/torch/nn/rnn.py

```diff
@@ -309,34 +280,24 @@ def _init_hidden(self, input):
                 h = h.share(*owners, crypto_provider=crypto_provider)
         return h
 
-    def _apply_time_step(self, x, h, c, t, reverse_direction=False):
+    def _apply_time_step(self, x, h, t, reverse_direction=False):
```

Comment:
Can you call the argument `hc` to be coherent with what you wrote above?

Reply:
Oops, yep.